styles/globals.css

```diff
@@ -167,3 +167,9 @@ code {
 .text-red-600 {
   color: #dc2626;
 }
+
+.sticky-header {
```

Comment:
@abhikhatri67 Can we handle it in tailwind itself? I have a reservation about putting code in global.css. We may get rid of this file soon.

Reply:
Sure, I will change it.